src/KeyVault/KeyVault/SecurityDomain/Cmdlets/InitSecurityDomainRecovery.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.Common.Strategies;
+using Microsoft.Azure.Commands.KeyVault.Properties;
+using Microsoft.Azure.Commands.KeyVault.SecurityDomain.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common;
+using Newtonsoft.Json;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.KeyVault.SecurityDomain.Cmdlets
+{
+    [Cmdlet(VerbsData.Initialize, ResourceManager.Common.AzureRMConstants.AzurePrefix + CmdletNoun.KeyVault + "SecurityDomainRecovery", SupportsShouldProcess = true, DefaultParameterSetName = ByName)]
+    [OutputType(typeof(bool))]
+    public class InitSecurityDomainRecovery : SecurityDomainCmdlet
+    {
+        [Parameter(HelpMessage = "Local file path to store the exported key.", Mandatory = true)]
+        [ValidateNotNullOrEmpty]
+        public string ExchangeKey { get; set; }
```

Comment:
Per my understanding, this parameter is to specify the location where stores initialized ExchangeKey. I'd suggest to use KeyOutPutPath or ExchangeKeyOutPutPath. We also need to consider whether supports outputting ExchangeKey as a plaintext.